lib/tests/function.rs

```diff
@@ -1036,6 +1036,26 @@ async fn function_array_union() -> Result<(), Error> {
 	Ok(())
 }
 
+#[tokio::test]
+async fn function_array_windows() -> Result<(), Error> {
+	let sql = r#"
+		RETURN array::windows([0, 1, 2, 3], 2);
+		RETURN array::windows([0, 1, 2], 2);
+		RETURN array::windows([0, 1, 2], 3);
+		RETURN array::windows([0, 1, 2, 3, 4, 5], 3);
```

Comment:
Can we check/decide what is the expected behaviour if the window is larger than the array?

```RETURN array::windows([0, 1, 2], 5);```